scripts/config_helpers.py

```diff
@@ -21,19 +71,56 @@ def config():
 def symlink(from_):
     """Symlink the config directory to the specified path"""
     from_ = Path(from_)
-    # iterate through all template files in the conf directory
-    for template in Path("conf/").glob("*.template"):
-        # check if non-template file exists
-        if not (real_name := Path(f"conf/{template.stem}")).exists():
+    # iterate through all files in the from_ directory
+    for file in from_.glob("*"):
+        # check if the file exists in the conf directory
+        if not (real_name := Path(f"conf/{file.name}")).exists() and not real_name.is_symlink():
             # if not, symlink to the file in the from_ directory
-            link_path = from_ / real_name.name
-            if not link_path.exists():
-                logger.error(f"Could not find {link_path}")
-                continue
             logger.warning(f"Symlinking {real_name} to {from_ / real_name}")
             real_name.symlink_to(from_ / real_name.name)
+
+
+@config.command()
+@click.option(
+    "--from",
+    "from_",
+    type=click.Path(exists=True, file_okay=False),
+    help="The path to the existing config directory",
+)
+@click.option(
+    "--strategy",
+    type=click.Choice(["ask", "overwrite", "preserve"]),
+    help="The strategy to use when merging",
```

Comment:
```suggestion
    help="The strategy to use when merging. Choose one from ask, overwrite, or preserve. If not provided then default is ask",
```

Reply:
I can't check right now, but are the options not presented by click in the help text?